main.rb

```diff
@@ -0,0 +1,47 @@
+require_relative 'app'
+
+def main
+  app = App.new
+
+  puts 'Welcome to School Library App!'
+  puts ' '
+
+  loop do
+    display_menu
+    option = gets.chomp
+
+    if option == '7'
+      puts 'Thank you for using this app!'
+      break
+    end
+    options(option, app)
+  end
+end
+
+def display_menu
+  puts 'Please choose an option by entering a number:'
+  puts '1 - List all books'
+  puts '2 - List all people'
+  puts '3 - Create a person'
```

Comment:
- [ ] Great job so far @Nuga0909 There are just a few changes I will suggest you implement and you'll be on your way to the next Milestone :+1: 
- [ ] Your project looks great so far kudos :clap: However, when I was trying out your console, when I tried creating a person, I encountered an error. As a result, I was unable to create either a student or teacher so kindly fix this :+1: 
- [ ] I was able to create a book. But because I was unable to create a person, I was unable to test out the other features of your console app so kindly ensure that `Create a rental` and `List all rentals for a given person ID` also work as expected. So the next reviewer can confirm that they all work seamlessly without errors :wink: 

- Video Reference

[Screencast from 11-10-2023 04:01:06 PM.webm](https://github.com/Nuga0909/oop-school-library/assets/102699958/3559f6ae-7040-47bf-8219-6a4e9fbb3ae9)

Reply:
ive fixed. I noticed the bug after  submission and fixed  it